fixed the security question of the FAQ (#823)
port-labs/port-docs: docs/faq/faq.md

## What security does Port have in place?

> We put a lot of thought into Port’s design to make it secure. As a result, Port’s product doesn’t store secrets or credentials, and doesn't require whitelisting of IPs. 

Reviewer: ```suggestion
We put a lot of thought into Port’s design to make it secure. Consequently, it doesn’t store secrets or credentials, and doesn't require whitelisting of IPs. 
```

> You can find our secured push-only architecture [here](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog)

Reviewer: ```suggestion
You can take a look at our secure, push-only architecture [here](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog).
```

> Security and privacy is top of mind at Port. We use industry standard encryption protocols, data is encrypted both at rest and in transit, with complete isolation between clients and data access logging and auditing. Port is SOC2 compliant, undergoes regular Pentests, product security and compliance review

Reviewer: ```suggestion
Security and privacy are a top priority at Port. We use industry-standard encryption protocols, data is encrypted both at rest and in transit, with complete isolation between clients and data access logging and auditing. Port is SOC2 compliant, and undergoes regular pentests, product security and compliance reviews.
```

> You may find the complete coverage of Port's security posture in our security [page](https://www.getport.io/security)

Reviewer: ```suggestion
You can find the complete coverage of Port's security policy in our security [page](https://www.getport.io/security).
```
